src/modules/gnd_pos_control/GroundRoverPositionControl.cpp

```diff
@@ -257,51 +262,33 @@ GroundRoverPositionControl::control_position(const matrix::Vector2f &current_pos
 			}
 		}
 
-		if (pos_sp_triplet.current.type == position_setpoint_s::SETPOINT_TYPE_IDLE) {
-			_att_sp.roll_body = 0.0f;
-			_att_sp.pitch_body = 0.0f;
-			_att_sp.yaw_body = 0.0f;
-			_att_sp.thrust_body[0] = 0.0f;
+		if (pos_sp_triplet.current.type == position_setpoint_s::SETPOINT_TYPE_IDLE ||
+		    pos_sp_triplet.current.type == position_setpoint_s::SETPOINT_TYPE_LOITER ||
```

Comment:
Sidenote, we'll probably want to keep loiter in some capacity. This is what's used when clicking on the screen to go to a position.